### PR TITLE
Pass custom_objects as args in TocoConverter.from_keras_model_file

### PR DESCRIPTION
**Pull request by benjamintanweihao**
Pass `custom_objects` as an argument to `TocoConverter.from_keras_model_file()` to allow for things like custom Keras layers.

### CODE REVIEW

File: tensorflow/contrib/lite/python/lite.py [hunk 1]
```diff
@@ -346,7 +346,8 @@ def from_keras_model_file(cls,
                             model_file,
                             input_arrays=None,
                             input_shapes=None,
-                            output_arrays=None):
+                            output_arrays=None,
```
Comment (gargn) [inline tensorflow/contrib/lite/python/lite.py]: 1. Add a description of the argument `custom_objects` to the docstring (follow the format of the other arguments).

2. Add a test function that uses `custom_objects` to the `FromKerasFile` class in `lite_test.py`. Make sure all of the tests in `FromKerasFile` in `lite_test.py` run. Feel free to use `--test_filter=FromKerasFile` or temporarily comment out the tests `testTFLiteGraphDef` and `testTFLiteGraphDefMissingShape` when you are testing locally since those tests currently don't work in open source.

Comment (gargn) [inline tensorflow/contrib/lite/python/lite.py]: 1. Add a description of the argument `custom_objects` to the docstring (follow the format of the other arguments).

2. Add a test function that uses `custom_objects` to the `FromKerasFile` class in `lite_test.py`. Make sure all of the tests in `FromKerasFile` in `lite_test.py` run. Feel free to use `--test_filter=FromKerasFile` or temporarily comment out the tests `testTFLiteGraphDef` and `testTFLiteGraphDefMissingShape` when you are testing locally since those tests currently don't work in open source.

**Review by gargn — CHANGES_REQUESTED** [review]

File: tensorflow/contrib/lite/python/lite.py (hunk 1 above)
Comment (benjamintanweihao) [reply tensorflow/contrib/lite/python/lite.py]: :+1: Give me a bit of time to work on it.

Comment (benjamintanweihao) [reply tensorflow/contrib/lite/python/lite.py]: @gargn @rmlarsen Done!

**Comment by tensorflowbutler** [comment]
Nagging Assignee @rmlarsen: It has been 23 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @rmlarsen: It has been 38 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @rmlarsen: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by rmlarsen** [comment]
@benjamintanweihao could you please resolve the conflicts?

**Comment by rthadur** [comment]
these files have been deleted in master , closing this PR
